traitsui/editors/tuple_editor.py

```diff
@@ -73,9 +58,10 @@ class ToolkitEditorFactory(EditorFactory):
     # 'enter_set' metadata or an editor defined.
     enter_set = Bool(False)
 
-#-------------------------------------------------------------------------
-#  'SimpleEditor' class:
-#-------------------------------------------------------------------------
+    # The validation function to use for the Tuple. If the edited trait offers
+    # already a validation function then the value of this trait will be
```

Comment:
This should be the other way around (unless you have a particular use-case in mind).  Because one `HasTraits` class can potentially have many different views, you generally want thing specified in the editor to override things specified in traits.

Although not exactly analogous, compare with (note that the `label` in the `Item` wins):
```
from traits.api import HasTraits, Float
from traitsui.api import View, Item


class Test(HasTraits):

    x = Float(label='Variable x')

    view = View(Item('x', label='Var x'))

t = Test()
t.edit_traits()
```